packages/renderer/src/lib/ui/NavItem.svelte

```diff
@@ -23,7 +24,7 @@ $: selected = meta.url === uri || (uri !== '/' && meta.url.startsWith(uri));
 
 const navItems: Writable<number> = getContext('nav-items');
 
-$: tooltipText = counter ? `${tooltip} (${counter})` : tooltip;
+$: tooltipText = counter ? (iconWithTitle ? `${counter}` : `${tooltip} (${counter})`) : tooltip;
```

Comment:
maybe we need to split into two lines as here it's starting to be complex to read